pkg/trafficpolicy/types.go

```diff
@@ -42,7 +42,7 @@ type HTTPRouteMatch struct {
 
 // TCPRouteMatch is a struct to represent a TCP route matching based on ports
 type TCPRouteMatch struct {
-	Ports []int `json:"ports:omitempty"`
+	Ports []uint32 `json:"ports:omitempty"`
```

Comment:
Can this be a uint16 instead, or was this done because Envoy protos use uint32?

Reply:
it was uint32 for envoy, but ya i can do uint16 and make the translation downstream.

I think that furthers the decoupling in a reasonable way

Reply:
Not a problem. It's fine.